packages/crypto/package.json

```diff
@@ -12,35 +12,21 @@
 		"access": "public"
 	},
 	"scripts": {
-		"prepublishOnly": "npm run build",
-		"prepare": "npm run build",
-		"prebuild": "npm run clean",
-		"build": "npm run build:protos && npm run build:compile",
-		"build:compile": "tsc -b tsconfig.json",
+		"build": "npm run build:protos && npm run build:cjs",
+		"build:cjs": "npx tsc -b ./tsconfig.json",
 		"build:protos": "npx pbjs src/keys.proto --ts src/proto.keys.ts",
-		"clean": "rimraf ./dist ./tsconfig.tsbuildinfo",
+		"clean": "npx rimraf ./dist ./tsconfig.tsbuildinfo",
 		"test:browser": "polendina --cleanup --timeout 60 src/**/*.spec.ts --webpack-config ../../webpack.test.js"
 	},
-	"repository": {
```

Comment:
uniform package.json setup, plus cleanup references. all package.json got this treatment